service-tests/dotnetstatus.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const Joi = require("joi");
+const ServiceTester = require("./runner/service-tester");
+
+const t = new ServiceTester({
+  id: "dotnetstatus",
+  title: "dotnet-status"
+});
+module.exports = t;
+
+t
+  .create("get nuget package status")
+  .get("/gh/json-api-dotnet/JsonApiDotNetCore/JsonApiDotNetCore.json")
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: Joi.equal("dependencies"),
+      value: Joi.equal(
+        "up to date",
+        "out of date",
+        "processing",
+        "project not found"
+      )
+    })
+  );
```

Comment:
Could you break out `"project not found"` into a separate test?

Reply:
done